client/src/components/Filters/FilterSalary.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import axios from "axios";
+import Slider from "@mui/material/Slider";
+import Box from "@mui/material/Box";
+
+export default function SalarySliderSteps({ handleSalary }) {
+  const [sliderValue, setSliderValue] = useState(0);
+
+  return (
+    <Box sx={{ width: 200, color: "green" }}>
```

Comment:
This should probably be wrapped in a label that specifies that it is a salary slider